src/main/java/LittleMissHelpful.java

```diff
@@ -0,0 +1,241 @@
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.IOException;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.util.ArrayList;
+
+public class LittleMissHelpful {
+    private static final String FILE_PATH = "./data/LittleMissHelpfullist.txt";
+    public static void main(String[] args) {
+        /**
+         * Simulates a chatbot
+         * Reads input and provides a reply
+         */
+
+        String name = "Ah Bang Mang";
+        String lineBreak = "---------------------------------";
+        String greeting = "Hello! I'm " + name + ".\nWhat you want sia?";
+        String exitLine = "Ok, I zao first then!";
+
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+
+        // Print greeting
+        System.out.println(lineBreak);
+        System.out.println(greeting);
+        System.out.println(lineBreak);
+
+        ArrayList<Task> list = new ArrayList<Task>();
+
+        // Load tasks from file
+        loadTasks(list);
+
+        int counter = list.size();;
+        while (true) { // Continuous input
```

Comment:
Would it be better to refactor this part into the the Parse class as suggested in the project brief week 3?

Reply:
agreed! Separating all the parsing commands into a separate parse class could make the code look neater

Reply:
Yes, thank you! I have abstracted out the classes now